lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -41,15 +42,16 @@ Function Get-MissingFeatures {
     }
 
     $missing_features = @($features | Where-Object InstallState -ne Installed)
-    
+
     return ,$missing_features # no, the comma's not a typo- allows us to return an empty array
 }
 
 Function Ensure-FeatureInstallation {
     # ensure RSAT-ADDS and AD-Domain-Services features are installed
 
-    Write-DebugLog "Ensuring required Windows features are installed..." 
+    Write-DebugLog "Ensuring required Windows features are installed..."
     $feature_result = Install-WindowsFeature $required_features
+    $result.reboot_required = $feature_result.RestartNeeded
```

Comment:
This is a change in how the module behaves, this should go into a separate PR.

Reply:
Same as with win_domain